repo/meta/schema/vX-repo-definitions.json

```diff
@@ -577,6 +585,14 @@
         "$ref": "#/dcosReleaseVersion",
         "description": "The minimum DC/OS Release Version the package can run on."
       },
+      "knownIssues": {
+        "type": "boolean",
+        "description": "Boolean field indicating if the package has any known issues."
+      },
+      "lastUpdated": {
```

Comment:
how is this timestamp calculated?

Reply:
@porridge has a ticket to update the release builder script in dcos-commons to calculate and embed the timestamp when making the universe PR. But I guess this is applicable for ceritifed packages only. If we want to add this field to ALL the packages, then we can probably modify the current CI to auto add this value when generating the universe file. Do you think that would be valuable ?

Reply:
We need to upgrade all packages with this field regardless if they're certified or not.
> we can probably modify the current CI to auto add this value when generating the universe file
this would be great. We should also have CI in universe that checks for this field and if it's not present it throws an error so all community package contributors are required to add this field.